gradio/components/chatbot.py

```diff
@@ -53,6 +53,7 @@ def __init__(
         latex_delimiters: list[dict[str, str | bool]] | None = None,
         rtl: bool = False,
         show_share_button: bool | None = None,
+        show_chat_copy_button: bool | None = None,
```

Comment:
Can just set to False by default
```suggestion
        show_chat_copy_button: bool = False,
```

Reply:
Can we name this argument just `show_copy_button` to make a little bit shorter.

Reply:
Agreed

Reply:
changed